app/templates/application.hbs

```diff
@@ -100,25 +96,25 @@
 
     <div class='links'>
     </div>
-</div>
+</nav>
 
 <form id='mobile-search' class='search' action='/search' {{ action "search" on="submit" }} >
-    <input type="text" class="search" name="q"
+    <input type="text" class="search" name="q" id="cargo-mobile-search"
             placeholder="Search"
             value={{searchQuery}}
             oninput={{action (mut searchQuery) value="target.value"}}
             autocorrect="off"
-            tabindex="1"
             required>
+    <label for="cargo-mobile-search">Search</label>
 </form>
 
-<div id="main" class='inner-content'>
+<main id="main" class='inner-content'>
```

Comment:
This is not ideal since each page should define it's own `main` - but I'm not familiar enough with the pages and don't have time to go through them all, so I think this is good enough for now.